static/app/components/replays/player/replayLoadingState.tsx

```diff
@@ -46,5 +46,6 @@ export default function ReplayLoadingState({
   if (readerResult.replayRecord?.is_archived) {
     return renderArchived ? renderArchived(readerResult) : <ArchivedReplayAlert />;
   }
+  // what about replay.hasProcessingErrors()?
```

Comment:
what about it?

Reply:
woo dealt with it. the component we have: `<ReplayProcessingError processingErrors={readerResult.replay.processingErrors()} />` doesn't even use that prop, and the message it shows is kind of long winded and stuff. So there's some cleanup to do. But at least now everywhere will be checking for all the cases by default!

Reply:
I updated the description with screen shots of the processingErrors case too.